config/locales/interface/input_elements/en_demand_buildings.yml

```diff
@@ -473,3 +473,37 @@ en:
         switches to hydrogen. The threshold value can be adjusted in the section 
         <a href=/scenario/flexibility/flexibility_net_load/demand-response-behavior-of-hybrid-heat-pumps
         >Demand response - behavior of hybrid heat pumps</a>.
+    buildings_space_heater_hybrid_crude_oil_heatpump_air_water_electricity_share:
+      title: Hybrid air heat pump (oil)
+      short_description:
+      description: |
+        This heater is a combination of an electric heat pump that draws its heat 
```

Comment:
I feel like the description for the HHP (or for all HHPs) is very long. I would prefer to keep the slider description short and to the point with relevant references to charts/other parts of the model, whereas the documentation may contain a more elaborate explanation of HHPs.